pkg/apis/workflow/v1alpha1/container_set_template_types.go

```diff
@@ -14,6 +14,7 @@ type ContainerSetTemplate struct {
 	VolumeMounts []corev1.VolumeMount `json:"volumeMounts,omitempty" protobuf:"bytes,3,rep,name=volumeMounts"`
 	// RetryStrategy describes how to retry a container nodes in the container set if it fails.
 	// Nbr of retries(default 0) and sleep duration between retries(default 0s, instant retry) can be set.
+	// The container won't retry if it's unable to locate the command.
```

Comment:
```suggestion
	// RetryStrategy describes how to retry container nodes if the container set fails.
	// Note that the container set will not be retried if a container's `command` cannot be located, as it will fail each time.
```

- This should match the other template types' `retryStrategy` [description](https://github.com/argoproj/argo-workflows/blob/24f791b9f5d9c3d946670366e1c86d802ea48ce1/pkg/apis/workflow/v1alpha1/workflow_types.go#L687)
- sleep and number of retries are described in the struct below and so are not necessary here
- be more specific as to what won't retry (it's the _entire_ container set)
  - also make that into a "Note" as it's more an aside than a key detail (and is not a spec nuance either)
  - same as above